elm-compiler.cabal

```diff
@@ -160,11 +160,13 @@ Executable elm
         CommandLineRouter.hs
 
     other-modules:
-        Elm.Compiler.Version
+        Elm.Compiler.Version,
+        Paths_elm_compiler
 
     Build-depends:
         base >=4.2 && <5,
         directory >= 1.0 && < 2.0,
+        elm-compiler,
```

Comment:
For some reason stack seems to fail unless an executable requires the corresponding library in every case. It appears to be more of a bug than a design decision, though it could be both.